lib/bigtable/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
Aaaand I just caught that as part of the publish script. Sorry for not reading the full PR before commenting :) We can probably still save on the `.gitignore` file if we use globbing in the root dir's .gitignore.